public/style.css

```diff
@@ -0,0 +1,75 @@
+body {
+    background-color: black;
+    color: white;
+    width: 1000px;
+    margin: 10px auto;
+    text-align: center;
+    padding: 10px;
+    user-select: none;
+  
+  }
+  
+  title {
+    font-size: 300px;
+  }
+  
+  h2 {
+    font-size: 75px;
+    text-align: center;
+    height: 75px;
+    font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+  }
+  
+  span{
+    font-size: 20px;
+    margin-top: 20px;
+  }
+  
+  div[id^="row-"] div {
+    width: 200px;
+    height: 200px;
+    border: 2px solid red;
+    float: left;
+    border-radius: 30px;
+  }
+  
+  div[id^="row-"] div:last-child {
+    border-right: none;
+  }
+  
+  div[id^="row-"] div:first-child {
+    border-left: none;
+  }
+  
+  div[id="row-1"] div{
+    border-top: none;
+  }
+  
+  div[id="row-3"] div{
+      border-bottom: none;
+  }
+  
+  div[id^="row-"] {
+    clear: float;
+    width: 608px;
+    box-sizing: border-box;
+    overflow: hidden;
+  }
```

Comment:
Great use of creating more specific CSS selectors so you didn't have to style each specific cell